src/views/Mutations.vue

```diff
@@ -39,7 +39,7 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 <script>
 import gql from 'graphql-tag'
-import { introspectionQuery, print } from 'graphql'
+import { getIntrospectionQuery as getGraphQLIntrospectionQuery, print } from 'graphql'
```

Comment:
Changed in GraphQL 15. Luckily they didn't remove the functionality.

Note that we already have a function in this module called `getIntrospectionQuery`, by coincidence. So if we imported like this, we could end up with weird build issues (e.g. recursively calling itself instead of calling the function from the `graphql` module). So I've renamed the import :+1: